src/reducers/walletConnectReducer.js

```diff
@@ -45,11 +45,72 @@ export type WalletConnectReducerState = {
   waitingRequest: ?string,
 };
 
-export type WalletConnectReducerAction = {
-  type: string,
-  payload: any,
+type SessionApproved = {
+  type: 'WALLETCONNECT_SESSION_APPROVED',
```

Comment:
I think we should use constants here in order to avoid mistyping

Reply:
Unfortunately, flow does not support using constants here, and they encourage this form: https://flow.org/en/docs/react/redux/#toc-typing-redux-actions. The only option is to use `typeof`, which results in `string`.

Using `string` here results in super weird error messages and flow forces you to even type empty arrays so it can choose the correct type.